resources/multisite_drupal_standard.make

```diff
@@ -1034,7 +1042,9 @@ projects[ec_resp][download][url] = https://github.com/ec-europa/ec_resp.git
 projects[ec_resp][download][tag] = 2.3.6
 
 projects[atomium][type] = theme
-projects[atomium][version] = 2.7
+projects[atomium][download][type] = git
+projects[atomium][download][url] = https://github.com/ec-europa/atomium.git
+projects[atomium][download][branch] = nept-1889
```

Comment:
Do not forget to change this part with:
`projects[atomium][version] = 3.0`

Reply:
It's not yet ready, this is why I'm using the branch, but I will as soon as Atomium 3.0 is ready.